cloudevents/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,109 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The Avro Turbo Format for CloudEvents defines how events are expressed in
+the [Avro 1.9.0 Specification][avro-spec].
+
+This differs from the [Avro format](avro-format.md) in that:
+
+- It is optimized for performance, preferring a more compact representation.
```

Comment:
*If* we work out a way to differentiate between string/URI/URI-ref attribute values, we should include that here too.